### ISSUE TITLE
Plaintext Emoticons/Emoji are not converted anymore

### ISSUE THREAD

**Issue by jbfriedrich** [overtake/TelegramSwift#418]
Since 5.9.2/5.9.3 Telegram for Desktop is not replacing plain text Emoji/Emoticons anymore. If you type ":)" or ":D" or ";)", for example, it is not replaced anymore with the matching Emoji equivalent. It seems to work for ":P" still, for whatever reason.

**Comment by overtake** [comment]
This feature will be returned in 5.9.4+. Sorry

**Comment by jbfriedrich** [comment]
Wow, that was quick. Thanks for the prompt answer. I just realized it is happening on the phone, too (iOS). Will that be fixed in a future version as well?

**Comment by fv0** [comment]
Now I’m always exposing myself when I’m at my Mac… gives it a kind of retro vibe with the old-school emojis. Anyway good to hear it’s coming back with 5.9.4. **;-)**

**Comment by YorikSar** [comment]
Looks like the issue still persists in 6.0

**Comment by bardonadam** [comment]
Issue still persists in 6.1.2

**Comment by overtake** [comment]
![image](https://user-images.githubusercontent.com/1260850/80707875-c1bc7800-8afb-11ea-9615-418b8a5dbe7e.png)
did you enable this setting?

**Comment by bardonadam** [comment]
Yes, but I just tried to actually send a message and it works. My bad. It was replacing it in the textfield during message composing before, if I remember correctly.

**Comment by matthewmorek** [comment]
The way this currently works is inconsistent between the main text input and the image caption/description input modes. Here are examples: 

#### Main text entry panel (chat)
![telegram-swift-emoji-input-main](https://user-images.githubusercontent.com/1137994/89514883-57883300-d7ce-11ea-9c56-82d28d13e961.gif)

#### Image text/description input modal
![CleanShot 2020-08-06 at 10 18 32](https://user-images.githubusercontent.com/1137994/89515040-830b1d80-d7ce-11ea-9ca3-dc52414a48bd.gif)

My personal preference is the **option two**:
- is way more predictable
- is less prone to user errors (you can see the effects of your typing immediately)
- you don't have to edit a message to correct a mistake
